Makefile

```diff
@@ -3,6 +3,17 @@
 # SPDX-License-Identifier: Apache-2.0
 
```

Comment:
You need to add ci to push the image  https://github.com/trustbloc/edge-agent/blob/master/ci/azure-pipelines.yml#L44

Reply:
done